netbox/extras/forms/filtersets.py

```diff
@@ -224,19 +225,29 @@ class WebhookFilterForm(NetBoxModelFilterSetForm):
 
     fieldsets = (
         (None, ('q', 'filter_id', 'tag')),
-        (_('Attributes'), ('content_type_id', 'http_method', 'enabled')),
+        (_('Attributes'), ('http_method',)),
+    )
+    http_method = forms.MultipleChoiceField(
+        choices=WebhookHttpMethodChoices,
+        required=False,
+        label=_('HTTP method')
+    )
+
+
+class EventRuleFilterForm(NetBoxModelFilterSetForm):
+    model = EventRule
+    tag = TagFilterField(model)
+
+    fieldsets = (
+        (None, ('q', 'filter_id', 'tag')),
+        (_('Attributes'), ('content_type_id', 'enabled')),
         (_('Events'), ('type_create', 'type_update', 'type_delete', 'type_job_start', 'type_job_end')),
     )
     content_type_id = ContentTypeMultipleChoiceField(
         queryset=ContentType.objects.filter(FeatureQuery('webhooks').get_query()),
```

Comment:
This will need to be updated for #14153 and to reference "event rules" as a feature.